dvc/commands/experiments/ls.py

```diff
@@ -16,10 +16,14 @@ def run(self):
             num=self.args.num,
             git_remote=self.args.git_remote,
         )
+        tags = self.repo.scm.describe(exps)
+        remained = {baseline for baseline, tag in tags.items() if tag is None}
+        branches = self.repo.scm.describe(remained, base="refs/heads")
+
         for baseline in exps:
-            tag = self.repo.scm.describe(baseline)
+            tag = tags[baseline]
             if not tag:
-                branch = self.repo.scm.describe(baseline, base="refs/heads")
+                branch = branches[tag]
                 if branch:
                     tag = branch.split("/")[-1]
```

Comment:
This split should be removed (same bug as https://github.com/iterative/dvc/issues/7933)

Reply:
It will make the output like:
```
refs/heads/main:
	exp-f88ff
```

Reply:
> It will make the output like:
> 
> ```
> refs/heads/main:
> 	exp-f88ff
> ```

We should only keep `/` for names from tags or branches.

Reply:
When you call `describe` to get the name you have to remove the `refs/.../` prefix and not rely on `split`.

Reply:
The output should be
```
main:
	exp-f88ff
```